cmd/subnetcmd/deploy.go

```diff
@@ -90,8 +90,15 @@ func deploySubnet(cmd *cobra.Command, args []string) error {
 
 	// get the network to deploy to
 	var network models.Network
-	if deployLocal {
-		network = models.Local
+	if deployNetwork != "" {
+		switch deployNetwork {
+		case strings.ToLower(models.Fuji.String()):
+			network = models.Fuji
+		case "local":
```

Comment:
there is also a modes.Local.String()

Reply:
Yes but that `String()` function prints `Local Network`. I think we just want people to enter `local`.

Reply:
If we take the approach I mentioned above, we can synchronize this with the model string, which I think would be preferable.

Reply:
With bool flags it's not even needed